tests/kafkatest/tests/core/downgrade_test.py

```diff
@@ -120,11 +120,11 @@ def test_upgrade_and_downgrade(self, version, compression_types, security_protoc
         self.setup_services(kafka_version, compression_types, security_protocol, static_membership)
         self.await_startup()
 
-        self.logger.info("First pass bounce - rolling upgrade")
+        print("First pass bounce - rolling upgrade", flush=True) # Force some stdout for Travis
```

Comment:
This seems weird. If we want to force stdout, we should configure the logged to do so, right?

Reply:
I tried to configure ```ducktape```  to force more output but ```ducktape``` supports only ```--debug```. It is too noisy to me.

Another way is to make a patch for ```ducktape``` for INFO level support. WDYT?

Reply:
That sounds good to me.

Reply:
https://github.com/confluentinc/ducktape/issues/260